### PR TITLE
Create 24.2 docs

### PR DESCRIPTION
**Pull request by llewellyn-sl**
Adds 24.2 docs as `/latest`. 
Removes Entra ID and service principals from 24.1 Azure CE page.

### CODE REVIEW

**Review by christopher-hakkaart — APPROVED** [review]
I've requested a couple of minor changes and added a few comments.

The links I checked look okay to me. But having a sanity review from someone who has done a version/latest update would be worthwhile.

The drop down navigation is a little buggy (won't go back to latest after I change it), but this might be a web thing rather than a PR thing. Is it just me?

Nothing blocking.

Minor comments that could be addressed in a separate PR:
- Wording around Azure Batch vs Batch. Azure docs refer to Batch, and I think we could too after the first use in a section.
- Unclear usage of platform in Azure docs. I assumed it was Platform but we could be more explicit.
- Examples using 'tower'. I assumed we were moving away from tower to help with branding cohesion. Maybe this is a hangover from a previous docs version.
- I won't approach the Seqera Platform vs Cloud vs Enterprise here

File: platform_versioned_docs/version-24.2/credentials/agent_credentials.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+[Tower Agent](../supported_software/agent/agent.mdx) enables Seqera Platform to launch pipelines on HPC clusters that do not allow direct access through an SSH client. Tower Agent authenticates a secure connection with Seqera using a Tower Agent credential.
+
+## Tower Agent sharing
+
+You can share a single Tower Agent instance with all members of a workspace. Create a Tower Agent credential, with **Shared agent** enabled, in the relevant workspace. All workspace members can then use this credential (Connection ID + Seqera access token) to use the same Tower Agent instance.
+
+## Create a Tower Agent credential
+
+1.  Add your credentials to your organization or personal workspace:
+    - From an organization workspace: Go to the **Credentials > Add Credentials**.
```
Comment (justinegeffen) [inline platform_versioned_docs/version-24.2/credentials/agent_credentials.mdx:17]:
```suggestion
    - From an organization workspace: Go to **Credentials > Add Credentials**.
```

File: platform_versioned_docs/version-24.2/credentials/aws_registry_credentials.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+1. Open the [IAM console](https://console.aws.amazon.com/iam/).
+2. Select **Users** from the navigation pane.
+3. Select the name of the user whose keys you want to manage, then select the **Security credentials** tab. We recommend creating an IAM user specifically for Wave authentication instead of using existing credentials with broader permissions.
+4. In the **Access keys** section, select **Create access key**. Each IAM user can have only two access keys at a time, so if the Create option is deactivated, delete an existing access key first.
+5. On the **Access key best practices & alternatives** page, select **Other** and then **Next**.
+6. On the **Retrieve access key** page, you can either **Show** the user's secret access key details, or store them by selecting **Download .csv file**.
+7. The newly created access key pair is active by default and can be stored as a container registry credential in Seqera.
+
+:::note
+Your credential must be stored in Seqera Platform as a **container registry** credential, even if the same access keys already exist as a workspace credential.
```
Comment (justinegeffen) [inline platform_versioned_docs/version-24.2/credentials/aws_registry_credentials.mdx:29]:
```suggestion
Your credential must be stored in Seqera as a **container registry** credential, even if the same access keys already exist as a workspace credential.
```

File: platform_versioned_docs/version-24.2/credentials/managed_identities.mdx [hunk 3]
```diff
@@ -0,0 +1,63 @@
+---
+title: "Managed identities"
+description: "Configure managed identities in Seqera Platform."
+date: "12 Jun 2024"
+tags: [organizations, administration, managed identities, clusters, ssh, credentials]
+---
+
+Managed identities offer significant advantages for High-Performance Computing (HPC) environments by enabling granular access control for individual workspace users. Unlike traditional SSH credentials that grant all workspace users access to HPC clusters using the same set of credentials, managed identities ensure each user’s activity is logged under their own credentials. This preservation of user identity is crucial as it naturally inherits the HPC system's fair usage queue policies, mitigates the noisy neighbor problem, and reduces the long wait times associated with First-In-First-Out (FIFO) queues common with shared SSH credentials.
```
Comment (justinegeffen) [inline platform_versioned_docs/version-24.2/credentials/managed_identities.mdx:8]:
```suggestion
Managed identities offer significant advantages for high performance computing (HPC) environments by enabling granular access control for individual workspace users. Unlike traditional SSH credentials that grant all workspace users access to HPC clusters using the same set of credentials, managed identities ensure each user’s activity is logged under their own credentials. This preservation of user identity is crucial as it naturally inherits the HPC system's fair usage queue policies, mitigates the noisy neighbor problem, and reduces the long wait times associated with First-In-First-Out (FIFO) queues common with shared SSH credentials.
```

File: platform_versioned_docs/version-24.2/data/datasets.mdx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+description: "Using datasets in Seqera Platform."
+date: "21 Apr 2023"
+tags: [datasets]
+---
+
+:::note
+This feature is only available in organization workspaces.
+:::
+
+Datasets in Seqera Platform are CSV (comma-separated values) and TSV (tab-separated values) files stored in a workspace. They are used as inputs to pipelines to simplify data management, minimize user data-input errors, and facilitate reproducible workflows.
```
Comment (justinegeffen) [inline platform_versioned_docs/version-24.2/data/datasets.mdx:12]:
```suggestion
Datasets in Seqera are CSV (comma-separated values) and TSV (tab-separated values) files stored in a workspace. They are used as inputs to pipelines to simplify data management, minimize user data-input errors, and facilitate reproducible workflows.
```

**Review by justinegeffen — APPROVED** [review]
A few minor suggestions for Seqera/Seqera Platform.

I did some reading around HPC and "high performance computing" seems to be the most commonly-used way of presenting it although "high-performance" is also used. I don't see any major changes needed anywhere else - we generally refer to "HPC" and not the long form anyway. 

Great job!

**Pull request merged**